Tests/TestPlans/Apollo-CITestPlan.xctestplan

```diff
@@ -13,6 +13,7 @@
   },
   "testTargets" : [
     {
+      "parallelizable" : true,
```

Comment:
I'm a little bit concerned about this causing some weird test failures. When doing tests in parallel, you have to be really careful about test isolation in a more specific way, which is a **good thing**, but I don't know if all of our tests are handling that correctly right now.

But if they are passing right now, then it's probably okay. Guess we'll find out!

Reply:
I can't say that I saw a massive improvement in time-to-run after parallelization was enabled, so I'm not opposed to disabling it if it begins to cause problems; I'd rather tackle the root cause of those anyways and enforce better test isolation.